crates/host/Cargo.toml

```diff
@@ -15,6 +15,8 @@ tracing = "0.1"
 parking_lot = "0.11.1"
 once_cell = "1.5.2"
 rand = "0.8.3"
+bimap = "0.6"
+plru = { git = "https://github.com/crdumoul/plru" }
```

Comment:
This is an unmaintained fork of a crate. It's concurrent map that doesn't specify how it handles locking. I don't know if that we want this as a dependency unless we are willing to fork it andd maintain it ourself?

Also why do we need a LRU? What is issue with instances not being evenly used? I don't think it matters which instance is used.